apps/frontend/src/app/Components/Character/CharacterCardPico.tsx

```diff
@@ -1,29 +1,35 @@
+import { CharacterKey } from '@genshin-optimizer/consts';
 import { characterAsset } from '@genshin-optimizer/g-assets';
 import { Box, CardActionArea, Skeleton, Typography } from '@mui/material';
 import { Suspense, useCallback } from 'react';
 import Assets from '../../Assets/Assets';
 import { getCharSheet } from '../../Data/Characters';
 import { ascensionMaxLevel } from '../../Data/LevelData';
+import { ElementIcon } from '../../KeyMap/StatIcon';
 import useCharacter from '../../ReactHooks/useCharacter';
 import useDBMeta from '../../ReactHooks/useDBMeta';
-import { CharacterKey } from '../../Types/consts';
+import { iconInlineProps } from '../../SVGIcons';
 import BootstrapTooltip from '../BootstrapTooltip';
 import CardDark from '../Card/CardDark';
+import ColorText from '../ColoredText';
 import ConditionalWrapper from '../ConditionalWrapper';
 import SqBadge from '../SqBadge';
-import StatIcon from '../StatIcon';
 
 export default function CharacterCardPico({ characterKey = "", index = -1, onClick }: { characterKey: CharacterKey | "", index?: number, onClick?: (characterKey: CharacterKey) => void }) {
   const { gender } = useDBMeta()
-  const teammateSheet = characterKey ? getCharSheet(characterKey, gender): undefined
+  const teammateSheet = characterKey ? getCharSheet(characterKey, gender) : undefined
   const character = useCharacter(characterKey)
   const onClickHandler = useCallback(() => characterKey && onClick?.(characterKey), [characterKey, onClick])
   const actionWrapperFunc = useCallback(
     children => <CardActionArea onClick={onClickHandler}>{children}</CardActionArea>,
     [onClickHandler])
   if (teammateSheet && character) {
     const title = <Suspense fallback={<Skeleton variant="text" width={100} />}>
-      <Typography>{teammateSheet.elementKey && StatIcon[teammateSheet.elementKey]} {teammateSheet.name}</Typography>
+      <Typography>
+        <ColorText color={teammateSheet.elementKey}>
+          {teammateSheet.elementKey && <ElementIcon ele={teammateSheet.elementKey} iconProps={iconInlineProps} />} {teammateSheet.name}

```

Comment:
I'm not a huge fan of coloring the name of the teammate. Element icon is fine though. But up to you

Reply:
I'll color the element icon